keyboards/keebio/quefrency/keymaps/rogthefrog/keymap.c

```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
```

Comment:
This is more of a nitpick, but we recommend enum's for layer names, as it is easier to maintain long term. 

```suggestion
enum layers {
    _BASE,
    _FN1
);
```